thirdparty/openldap/zimbra-openldap/rpm/SPECS/openldap.spec

```diff
@@ -23,22 +21,10 @@ URL:                http://www.openldap.org
 The Zimbra openldap build
 
 %changelog
-* Fri Sep 9 2016  Zimbra Packaging Services <packaging-devel@zimbra.com> - VERSION-1zimbra8.7b9ZAPPEND
-- Another upstream fix for ITS#8448, ITS#8460, ITS#8462, ITS#8490, ITS#8493
-* Thu Jul 29 2016  Zimbra Packaging Services <packaging-devel@zimbra.com> - VERSION-1zimbra8.7b8ZAPPEND
-- Import fix from upstream for ITS#8448, ITS#8460, ITS#8462.
-* Thu Jun 9 2016  Zimbra Packaging Services <packaging-devel@zimbra.com> - VERSION-1zimbra8.7b7ZAPPEND
-- Finalized patch for ITS#8432
-* Thu Jun 9 2016  Zimbra Packaging Services <packaging-devel@zimbra.com> - VERSION-1zimbra8.7b6ZAPPEND
-- syncrepl patch for ITS#8432
-* Thu Jun 9 2016  Zimbra Packaging Services <packaging-devel@zimbra.com> - VERSION-1zimbra8.7b5ZAPPEND
-- syncprov patch for ITS#8432
-* Thu Jun 9 2016  Zimbra Packaging Services <packaging-devel@zimbra.com> - VERSION-1zimbra8.7b4ZAPPEND
-- Add debug logging for ITS#8432
-* Wed Apr 27 2016  Zimbra Packaging Services <packaging-devel@zimbra.com> - VERSION-1zimbra8.7b3ZAPPEND
-- Add patch for ITS#8413
-* Thu Feb 11 2016  Zimbra Packaging Services <packaging-devel@zimbra.com> - VERSION-1zimbra8.7b2ZAPPEND
-- Add patch for ITS#7506
+* Tue Apr 24 2018  Zimbra Packaging Services <packaging-devel@zimbra.com> - VERSION-1zimbra8.7b1ZAPPEND
```

Comment:
Changing release from b9 to b1 is downgrading the package. 
We can use either b10 or 1zimbra8.8b1ZAPPEND

Reply:
@Prashantsurana Incorrect.